### ISSUE TITLE
KeePass importer ignore Notes

### ISSUE THREAD

**Issue by maky** [nextcloud/passman#430]
### Steps to reproduce
1. Export KeePass paswords in CSV 1.x version
2. Import into PassMan
3. Notes are not imported.

### Expected behaviour
Notes are the last field in the CSV file, but seems to be they are ignored

### Configuration
**Operating system**:   Owncloud 10.0.4.4

**Browser**:      
Chrome

**Extensions that might cause interference**:   

**Passman version**:    
2.1.4

   
#### Browser log
<details>
<summary>Browser log</summary>
```
Insert your browser log here, this could for example include:

a) The javascript console log
b) The network log
c) ...
```
</details>   

#### Cloud log

<details>
<summary>Cloud log</summary>

```
Insert your cloud log here
```
</details>

## Feature request

**User type**: Logged-in or public user

**User level**: Beginners, intermediate or advanced

### Sponsorship

 
**Are you a developer willing to implement this feature?**: yes/no

**Can you sponsor the development of this feature or do you know someone who can?**: yes/no

**Comment by dennison-williams-stjames** [comment]
I confirmed this bug

**Comment by animalillo** [comment]
Should be fixed on [2.3.7](https://github.com/nextcloud/passman/releases/tag/2.3.7)
